g2o/apps/g2o_simulator/simulator2d_segment.cpp

```diff
@@ -52,18 +52,18 @@ int main(int argc, char** argv) {
   bool hasGPS;
 
   int segmentGridSize;
-  double minSegmentLenght, maxSegmentLenght;
+  double minSegmentLength, maxSegmentLength;
 
   std::string outputFilename;
   arg.param("simSteps", simSteps, 100, "number of simulation steps");
   arg.param("nLandmarks", nlandmarks, 1000, "number of landmarks");
   arg.param("nSegments", nSegments, 1000, "number of segments");
   arg.param("segmentGridSize", segmentGridSize, 50,
             "number of cells of the grid where to align the segments");
-  arg.param("minSegmentLenght", minSegmentLenght, 0.5,
```

Comment:
Note that this change to the argument name may run the risk of breaking something. Let me know if it should be dropped.